app/controllers/audits_controller.rb

```diff
@@ -10,6 +10,15 @@ def show
     next_item
   end
 
+  def next
+    if next_item
+      redirect_to content_item_audit_path(next_item, filter_params)
+    else
+      flash.alert = "Unable to find the next item to audit."
```

Comment:
I would be more positive about this message. `There is not more items to audit`? Can you please check with Joe?